package.json

```diff
@@ -37,7 +38,7 @@
   },
   "scripts": {
     "start": "react-scripts start",
-    "build": "react-scripts build",
+    "build": "REACT_APP_API_URL='https://w6pkliozjh.execute-api.us-east-1.amazonaws.com/prod' react-scripts build",
```

Comment:
FYI you can add this environment variable to `.env.production.local` and you won't have to introduce code changes every time the production url changes.

Reply:
I want to set it as a default for people when they run the build script, even if they haven't yet configured their environment. It's an easy mistake to make.